src/components/app/App.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+import "../../shared/styles/main.css";
+import "./App.css";
+import Typeahead from "../typeahead/typeahead";
+import GitHubService from "../../services/github.service";
+import { GitHubRepo } from "../../shared/models/github";
+
+function App() {
+  const [data, setData] = useState<GitHubRepo[]>([]);
+  const githubService = new GitHubService();
+
+  useEffect(() => {
```

Comment:
Do you need to add the list of dependencies? Otherwise this useEffect will be run ever time component is rendered.

Reply:
You are correct. Forgot to add it in.